### PR TITLE
[WIP]Update script http 006

### PR DESCRIPTION
**Pull request by istoimenova**

### CODE REVIEW

**Comment by istoimenova** [comment]
@dboltovskyi, please review.

**Comment by istoimenova** [comment]
[WIP] - Clarified with Igor Kovalenko

File: test_scripts/Polices/build_options/006_ATF_P_TC_PTU_Trigger_IGN_Cycles_HTTP.lua [hunk 1]
```diff
@@ -205,7 +205,7 @@ function Test:TestStep_Register_app()
       self.mobileSession:ExpectResponse(correlationId, { success = true, resultCode = "SUCCESS" })
       self.mobileSession:ExpectNotification("OnHMIStatus", {hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN"})
     end)
-  EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "HTTP"})
+  EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "LOCK_SCREEN_ICON_URL"}, {requestType = "HTTP"}):Times(2)
```
Comment (dboltovskyi) [inline test_scripts/Polices/build_options/006_ATF_P_TC_PTU_Trigger_IGN_Cycles_HTTP.lua]: @istoimenova, 
1) In order to have OnSystemRequest with requestType = "LOCK_SCREEN_ICON_URL" appropriate type of requests should be allowed by policies
Currently in "files/jsons/Policies/Policy_Table_Update/ign_cycle_ptu.json" file such type is not allowed.

2) In this script order of OnSystemRequests are the following: "HTTP" and then "LOCK_SCREEN_ICON_URL"

Please make corrections

Comment (istoimenova) [inline test_scripts/Polices/build_options/006_ATF_P_TC_PTU_Trigger_IGN_Cycles_HTTP.lua]: @dboltovskyi, done in 3813a94d55441a8a692755c5e08030f5290829ce

**Pull request merged**